docs/tutorial.md

```diff
@@ -595,6 +598,33 @@ You must reconfigure any `image` resource definitions in your `PipelineResources
   example configuration at <https://github.com/mgreau/tekton-pipelines-elastic-tutorials>.
 - To learn more about obtaining logs, see [Logs](logs.md).
 
+### Prerequisites: Minikube
+
+Complete these prerequisites to run this tutorial locally using Minikube:
+
+- Install the [required tools](https://github.com/tektoncd/pipeline/blob/master/DEVELOPMENT.md#requirements).
+- Install [minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/) and start a sesion as follows:
+```bash
+minikube start --memory 10240 --cpus 6 
+```
+- Point your shell to minikube's docker-daemon by running `eval $(minikube -p minikube docker-env)`
+- Set up a registry on minkube by running `minikube addons enable registry` and `minikube addons enable registry-aliases`
+
+### Reconfigure `image` resources
+
+You must reconfigure any `image` resource definitions in your `PipelineResources` as follows:
+
+- Set the URL to `example.com/<image_name>`
```

Comment:
@psschwei I am not able to understand instructions in this section, what is this URL meant for? we enabled registry in the previous section. Also, since we are using `-L` flag, no need to setup `KO` related env (i.e. unlike `KO_DOCKER_REPO` for docker for desktop). What would this `example.com` be in a `minikube` installation?

Reply:
`example.com` is one of the aliases created automatically for the registry via the `registry-aliases` addon (others are `example.org`, `test.com`, and `test.org`): 

```
docker@minikube:~$ cat /etc/hosts
127.0.0.1	localhost
::1	localhost ip6-localhost ip6-loopback
fe00::0	ip6-localnet
ff00::0	ip6-mcastprefix
ff02::1	ip6-allnodes
ff02::2	ip6-allrouters
172.17.0.3	minikube
172.17.0.1	host.minikube.internal
172.17.0.3	control-plane.minikube.internal
10.104.170.3	example.org
10.104.170.3	example.com
10.104.170.3	test.com
10.104.170.3	test.org
```

If we want to be able to push/pull to the minikube Docker registry, we'll need to use one of its aliases. If we don't, it will try to resolve to Docker Hub -- this is the error I got when running the `build-docker-image-from-git-source-task-run` TaskRun using just `leeroy-web` as the URL for the `skaffold-image-leeroy-web` PipelineResource:

```
error checking push permissions -- make sure you entered the correct tag name, and that you are authenticated correctly, and try again: checking push permission for "leeroy-web": POST https://index.docker.io/v2/library/leeroy-web/blobs/uploads/: UNAUTHORIZED: authentication required; [map[Action:pull Class: Name:library/leeroy-web Type:repository] map[Action:push Class: Name:library/leeroy-web Type:repository]]
```

@pritidesai I can add some clarifying remarks to the `Reconfigure `image` resources` section to explain why we're using `example.com` as the registry alias here. While we could use an alternative hostname (for example, `host.docker.local` like in the Docker for Desktop example), it would require editing `/etc/hosts` and the `CoreDNS` configmap, so I assumed the simpler option was better. :)

Reply:
Thanks @psschwei for the explanation, you can tell how much I know `minikube` configuration 🤦‍♀️

This looks good, no need to add alternative hostname. It will be great to clarify why `example.com` and adding may be this command to get the list of available hostnames ` minikube ssh -- cat /etc/hosts`

Reply:
No worries -- I only learned this in the course of doing this PR (and from a very helpful suggestion from @kameshsampath ). Will make the update shortly.

Reply:
Update made